Tests/ML/histopathology/test_datamodule_caching.py

```diff
@@ -0,0 +1,166 @@
+import shutil
```

Comment:
Can we respect the same hierarchy of folders in the tests? it's easier to locate them. Not sure if it's a commit issue - this is something I fixed on the RadiomicsNN front about a week ago

Reply:
Yes of course. Updated